src/parser/resource.rs

```diff
@@ -470,7 +477,7 @@ pub fn build_resources_recursively(
                         Box::new(third_obj),
                     )
                 }
-                "!Ref" | "Ref" => {
+                Some("Ref") => {
```

Comment:
I believe !Ref is legit as well: https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/intrinsic-function-reference-ref.html#ref-example

Reply:
Ah yes... I made a boo-boo here... Because I changed approaches in the middle of doing this and since those aren't `Fn::` intrinsics I forgot about them...